build-a-game-playing-agent/agent_test.py

```diff
@@ -125,8 +122,7 @@ def score(self, game, player):
 
 
 def makeEvalStop(limit, timer, value=None):
-
-    class EvalStop():
+    class EvalStop:
```

Comment:
I'm not too familiar with Python 3 formatting rules, but either syntax is acceptable Python 2 code.